lib/syntax.ml

```diff
@@ -10,14 +10,13 @@ open Loc
 
 (* association list of pragmas *)
 
-type pragma = [`NestedProtocols | `ShowPragmas | `PrintUsage]
-[@@deriving show]
+type pragma = NestedProtocols | ShowPragmas | PrintUsage [@@deriving show]
```

Comment:
I kinda liked the open variant, but I can't really defend why using them is needed, so I guess this is not a problem.